client/me/billing-history/transactions-header.jsx

```diff
@@ -179,16 +179,16 @@ class TransactionsHeader extends React.Component {
 	renderAppsPopover() {
 		const { appFilters, filter, translate } = this.props;
 		const selectedFilter = find( appFilters, { value: filter.app } );
-		const selectedText = selectedFilter ? selectedFilter.title : translate( 'All Apps' );
+		const selectedText = selectedFilter ? selectedFilter.title : translate( 'All apps' );
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.